benchmark/common.js

```diff
@@ -52,12 +52,13 @@ Benchmark.prototype._parseArgs = function(argv, configs) {
       // Infer the type from the config object and parse accordingly
       const isNumber = typeof configs[match[1]][0] === 'number';
       const value = isNumber ? +match[2] : match[2];
-      cliOptions[match[1]] = [value];
+      if (!cliOptions[match[1]]) cliOptions[match[1]] = [];
```

Comment:
>It shouldn't need braces.

Really? I think I've always seen single-line conditional bodies wrapped in braces in node's codebase, so I assumed that's our style.

>As far as a variable name goes, I don't really have a preference.

If you don't have a preference, I'll go with `config` instead.